msteams-platform/m365-apps/prerequisites.md

```diff
@@ -42,70 +42,70 @@ You also need to enable sideloading for your tenant:
 
  1. Turn on **Upload custom apps**, and select **Save**.
 
-## Enroll your developer tenant for Office 365 Targeted releases
+## Enroll your developer tenant for Microsoft 365 Targeted releases
 
 > [!IMPORTANT]
-> It can take up to five days after creating a [Microsoft 365 developer sandbox tenant](/office/developer-program/microsoft-365-developer-program-get-started) and enrolling in [Office 365 Targeted releases](#enroll-your-developer-tenant-for-office-365-targeted-releases) for sideloaded Teams apps to appear in Outlook and Office.
+> It can take up to five days after creating a [Microsoft 365 developer sandbox tenant](/office/developer-program/microsoft-365-developer-program-get-started) and enrolling in [Microsoft 365 Targeted releases](#enroll-your-developer-tenant-for-office-365-targeted-releases) for sideloaded Teams apps to appear in Outlook and Microsoft 365.
 
-To enroll your test tenant for Office 365 targeted releases:
+To enroll your test tenant for Microsoft 365 targeted releases:
 
 1. Sign in to [Microsoft 365 admin center](https://admin.microsoft.com) with your test tenant credentials.
-1. Go to **Settings** > **Org Settings** > **Organization profile**.
-1. Select **Release preferences**.
-1. Select any *Targeted release* preference:
+2. Go to **Settings** > **Org Settings** > **Organization profile**.
```

Comment:
It looks funny in source, but its MS Docs best practice to do ordered lists with every item being "1." (it auto-iterates at build time and removes chance of human off-by-one errors)